### PR TITLE
Update Master Arbitration Updates to clarify that (device_id, role, election_id) triples are only unique for live controllers.

### PR DESCRIPTION
**Pull request by jonathan-dilorenzo**
Re-opening of #368 on a local branch.

See issue #365 for more details. I also made some minor language (and formatting?) improvements.

### CODE REVIEW

File: docs/v1/P4Runtime-Spec.mdk [hunk 1]
```diff
@@ -724,32 +724,35 @@ follows:
...
+     and `role`, as described in a
+     [later section](#sec-arbitration-notification); and
+
+  2. The P4Runtime server will be without a primary controller, until a client
+     sends a successful `MasterArbitrationUpdate` (as per the rules in a
+     [later section](#sec-arbitration-updates)).
+
+* The mechanism through which the controller receives the P4Runtime server
+  details are implementation specific and beyond the scope of this
+  specification. This includes includes the `device_id`, `ip` and `port`, as
```
Comment (antoninbas) [inline docs/v1/P4Runtime-Spec.mdk:741]: typo: extra "includes"

**Review by antoninbas — COMMENTED** [review]
LGTM, except for one small typo, but I'll let @smolkaj confirm that this is what he had in mind.

**Review by smolkaj — APPROVED** [review]
LGTM to me, thanks Jonathan!

File: docs/v1/P4Runtime-Spec.mdk (hunk 1 above)
Comment (jonathan-dilorenzo) [reply docs/v1/P4Runtime-Spec.mdk:741]: Alas, you merged without me fixing the typo. Is there an easy way to just quickly do this, or should I submit another PR?

Comment (antoninbas) [reply docs/v1/P4Runtime-Spec.mdk:741]: Apologies, I must have been distracted. Please fix it in a separate PR or even in your other pending PR

Comment (jonathan-dilorenzo) [reply docs/v1/P4Runtime-Spec.mdk:741]: Done! Thanks!

**Pull request merged**